### ISSUE TITLE
Customizable Deviceinfo

### ISSUE THREAD

**Issue by Squirrelsht** [disdain13/PokeRoadie#116]
I'm not sure if it's been mentioned from what I could see. My idea is simple in theory. Similar to how you can configure settings for destinations, pokemonstonotcatch etc., would it be difficult to make a deviceinfo xml to configure for a specific device? In my case, my galaxy s4 Mini. Instead of using the premade devices from the given list, i could (in theory) edit the deviceinfo xml with all the specifics for my device, and be all set.

**Comment by ghost** [comment]
@Squirrelsht what is the purpose of this? does it make any difference?

**Comment by Squirrelsht** [comment]
The purpose is for me to be able to enter my own device info. Rather than using preset devices.

**Comment by ghost** [comment]
obviously and that's the same thing you said up there.. but that's not what i asked though..

**Comment by xafelld** [comment]
I think he is wanting to make them think he is always playing from his device regardless of whether he is using PokeRoadie or his phone.

IF they are tracking the number of different devices used to access the same account, and the implementation of custom device info provides the servers all the same fields as the official app does, in the same way the app does, This could be one less flag they could use to spot irregularities. .

I am Pretty sure the limits of the available 3rd party API's make using anything but the official App pretty obvious  but having this functionality could be a step in the right direction, especially as the unofficial API's evolve.

**Comment by Squirrelsht** [comment]
Correct. That is the exact reason. Couldn't have said it any better.

**Comment by ArthurJS** [comment]
Apple device profiles would be neat, too
